cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -203,6 +208,22 @@ pScript = ScriptFile <$> Opt.strOption
   <> Opt.completer (Opt.bashCompleter "file")
   )
 
+pPaymentScript :: Parser ScriptFile
+pPaymentScript = ScriptFile <$> Opt.strOption
+  (  Opt.long "payment-script-file"
```

Comment:
Rather than having multiple parsers, lets turn it into a function that accepts two strings

Reply:
I made the requested changes, although looking back it it, I feel it makes the code less regular.

Reply:
I agree, I held off for a while doing this but in my Alonzo PR I had to because the number of script file parsers with minor changes to the help text was becoming ridiculous.